### ISSUE TITLE
[Bug] No avatar for Lumine

### ISSUE THREAD

**Issue by soleilvermeil** [SoleilVermeil/court-of-fontaine#6]
**What happened?**
When I went on `103509533`'s page, there was no avatar showing for this player.

**Additional context**
The image that showed: `https://enka.network/ui/UI_AvatarIcon_Traveler.png`
The image supposed to appear (according to Enka.Network): `https://enka.network/ui/UI_AvatarIcon_PlayerGirl.png`

Note that the API for this player has the following element:
```json
{
    ...
    "playerInfo": {
        "profilePicture": {
            "avatarId": 10000007
        },
        "showAvatarInfoList": [
            {
                "avatarId": 10000047,
                "level": 90
            }
        ],
        ...
    },
    ...
}
```

**Comment by soleilvermeil** [comment]
No avatar for Yae either.

**Comment by soleilvermeil** [comment]
No avatar for Raiden either.

**Comment by soleilvermeil** [comment]
> No avatar for Yae either.

Should be fixed in 118409a2509f92b067173a124376556e1eabd379.

> No avatar for Raiden either.

The source of the issue has been identified. Two types of avatars exist for Raiden:
* Players with the following (for example `719970581`) present no issue since 118409a2509f92b067173a124376556e1eabd379:
  ```json
  "profilePicture": {
      "id": 3900
  }
  ```
* But players with the following (for example `700583214`) have still a missing image:
  ```json
  "profilePicture": {
      "avatarId": 10000052
  }
  ```
  This issue has been adressed in 442ca5e3abecd48acbc8c2ca94210e82be429b26. Altough the fix is pretty gross and a more definitive solution bay me implemented later.
